{\rtf1\ansi\ansicpg1252\cocoartf1504\cocoasubrtf830
{\fonttbl\f0\fnil\fcharset0 AndaleMono;}
{\colortbl;\red255\green255\blue255;\red47\green255\blue18;\red0\green0\blue0;\red180\green36\blue25;
}
{\*\expandedcolortbl;;\cssrgb\c15686\c99608\c7843;\csgray\c0\c90000;\cssrgb\c76410\c21699\c12524;
}
\margl1440\margr1440\vieww10800\viewh8400\viewkind0
\pard\tx560\tx1120\tx1680\tx2240\tx2800\tx3360\tx3920\tx4480\tx5040\tx5600\tx6160\tx6720\pardirnatural\partightenfactor0

\f0\fs24 \cf2 \cb3 \CocoaLigature0 Williams-MBP:~ WilliamDean$ pip install Django\
Collecting Django\
  Using cached Django-1.11.6-py2.py3-none-any.whl\
Requirement already satisfied: pytz in /System/Library/Frameworks/Python.framework/Versions/2.7/Extras/lib/python (from Django)\
Installing collected packages: Django\
\cf4 Exception:\cf2 \
\cf4 Traceback (most recent call last):\cf2 \
\cf4   File "/Library/Python/2.7/site-packages/pip-9.0.1-py2.7.egg/pip/basecommand.py", line 215, in main\cf2 \
\cf4     status = self.run(options, args)\cf2 \
\cf4   File "/Library/Python/2.7/site-packages/pip-9.0.1-py2.7.egg/pip/commands/install.py", line 342, in run\cf2 \
\cf4     prefix=options.prefix_path,\cf2 \
\cf4   File "/Library/Python/2.7/site-packages/pip-9.0.1-py2.7.egg/pip/req/req_set.py", line 784, in install\cf2 \
\cf4     **kwargs\cf2 \
\cf4   File "/Library/Python/2.7/site-packages/pip-9.0.1-py2.7.egg/pip/req/req_install.py", line 851, in install\cf2 \
\cf4     self.move_wheel_files(self.source_dir, root=root, prefix=prefix)\cf2 \
\cf4   File "/Library/Python/2.7/site-packages/pip-9.0.1-py2.7.egg/pip/req/req_install.py", line 1064, in move_wheel_files\cf2 \
\cf4     isolated=self.isolated,\cf2 \
\cf4   File "/Library/Python/2.7/site-packages/pip-9.0.1-py2.7.egg/pip/wheel.py", line 345, in move_wheel_files\cf2 \
\cf4     clobber(source, lib_dir, True)\cf2 \
\cf4   File "/Library/Python/2.7/site-packages/pip-9.0.1-py2.7.egg/pip/wheel.py", line 316, in clobber\cf2 \
\cf4     ensure_dir(destdir)\cf2 \
\cf4   File "/Library/Python/2.7/site-packages/pip-9.0.1-py2.7.egg/pip/utils/__init__.py", line 83, in ensure_dir\cf2 \
\cf4     os.makedirs(path)\cf2 \
\cf4   File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/os.py", line 157, in makedirs\cf2 \
\cf4     mkdir(name, mode)\cf2 \
\cf4 OSError: [Errno 13] Permission denied: '/Library/Python/2.7/site-packages/django'\cf2 \
Williams-MBP:~ WilliamDean$ \
\
//The above shows what is returned when I installed Django again. //\
\
//There appears be no difference between pip freeze and pip list. // \
\
//ls shows me what is on my desktop as well as past/deleted items. //\
}